docs/middleware.rst

```diff
@@ -79,11 +94,19 @@ have the following priorities:
 - ``ExceptionInterfaceMiddleware``: 0
 - ``MessageInterfaceMiddleware``: 0
 - ``ModulesMiddleware``: 0
-- ``ProcessorMiddleware``: -250 (this middleware should always be at the end of
-  the chain)
 - ``RequestInterfaceMiddleware``: 0
 - ``SanitizerMiddleware``: -255 (this middleware should always be the last one)
 - ``UserInterfaceMiddleware``: 0
+- ``SanitizeHttpBodyMiddleware``: -200 (this middleware should always be after
```

Comment:
Should we sort them by priority?

Reply:
I've sorted the list by middleware priorities (should I sort from the first one to the last one or viceversa?) and I've added the `ContextMiddleware` as many times as it's really used taking care of describing which context is filled